docs/source/integrations/spark.rst

```diff
@@ -59,26 +65,37 @@ DataFrame containing the data from the file.
 .. code-block:: python
 
     from pyspark.sql.types import StructType, StructField, FloatType, StringType
-    schema = SparkStruct(
-        StructField(name=”sepal_length”, FloatType(), False),
-        StructField(name=”sepal_width”, FloatType(), False),
-        StructField(name=”petal_length”, FloatType(), False),
-        StructField(name=”petal_width”, FloatType(), False),
-    )
-    df = spark.read.csv("https://docs.bentoml.org/en/latest/_static/examples/batch/input.csv")
+    import urllib.request
+
+    urllib.request.urlretrieve("https://docs.bentoml.org/en/latest/_static/examples/batch/input.csv", "input.csv")
```

Comment:
I think this is necessary because we don't have directory listing for the docs server.